solutions/Sorting and Searching/traffic-lights.cpp

```diff
@@ -30,6 +30,10 @@ int main()
 		lengths.insert(right - point);
 
 		// insert the new point
+		/*
+		we can use this also which is easy to understand as set stores the value in ascending order
+		points.insert(point);
+		*/
 		points.insert(it, point);
```

Comment:
Thanks for the PR @vineet4571!
You can go ahead and remove `points.insert(it, point);` and add `points.insert(point);` instead :)